DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/traffic/ixnetwork/ixnetwork_ixia_client_impl.py

```diff
@@ -323,9 +323,14 @@ def set_ethernet_traffic(self, device, name, pkt_data, traffic_type):
             ti.Enabled = True
             for ep in range(ep_count):
                 config_element = ti.ConfigElement.find(EndpointSetId=ep + 1)
-                # set the rate
+                frame_rate_type = {
+                    "line_rate": "percentLineRate",
+                    "bps_rate": "bitsPerSecond",
+                    "ipg_rate": "interPacketGap",
+                    "pps_rate": "framesPerSecond"}
                 config_element.FrameRate.update(
-                    Type="framesPerSecond", Rate=pkt_data.get("rate", "100")
+                    Type=frame_rate_type[pkt_data.get("frame_type", "framesPerSecond")],
```

Comment:
and I think that `frame_type` should be changed to `rate_type` or `stream_rate_type` or `frame_rate_type`